todolist/settings.py

```diff
@@ -62,12 +62,12 @@
 # https://docs.djangoproject.com/en/1.7/ref/settings/#databases
 DATABASES = {
     'default': {
-        'ENGINE': 'mysql.connector.django',
-        'NAME': 'app_db',
-        'USER': 'app_user',
-        'PASSWORD': '1234',
-        'HOST': 'mysql',  # You can use a different host if your MySQL server is on a remote machine.
-        'PORT': '',  # Leave this empty to use the default MySQL port (3306).
+        'ENGINE': os.environ.get('ENGINE', 'django.db.backends.mysql'),
+        'NAME': os.environ.get('NAME', 'app_db'),
+        'USER': os.environ.get('USER', 'app_user'),
+        'PASSWORD': os.environ.get('PASSWORD', '1234'),
```

Comment:
The `PASSWORD` for the database is hardcoded. It's recommended to use environment variables or a secure method to manage sensitive information like passwords.